src/main.js

```diff
@@ -435,7 +435,7 @@ Formsy.Form = React.createClass({
   render: function () {
 
     return (
-      <form {...this.props} onSubmit={this.submit}>
+      <form id={this.props.id} {...this.props} onSubmit={this.submit}>
```

Comment:
I just noticed that the npm version downloaded is different from the version in the repository. That version does not have {...props}. Just so you know.